### ISSUE TITLE
Accept classnames

### ISSUE THREAD

**Issue by dimitrisfasoulas** [Mastermindzh/react-cookie-consent#15]
Styling through styles is nice but for bigger apps, you might already have classes that define styles for your elements.

It would be nice if we could pass class names for main container, content container, and button.

**Comment by Mastermindzh** [comment]
Agreed. What about this:

if you want to pass classes you use:

```js
<CookieConsent 
  disableStyles={true}
  classes="class 1 class2"
....
/>
```

I could have that done today.

**Comment by dimitrisfasoulas** [comment]
Looks good. I would prefer if there are different props for different elements though. So 
```
<CookieConsent 
  disableStyles={true}
  containerClasses="class1 class2"
  contentClasses="class1 class2"
  buttonClasses="class 1 class2"
....
/>
```

**Comment by Mastermindzh** [comment]
Makes sense, thinking of bootstrap you'd probably want to pass `btn btn-primary` to the button and something completely different to the bar itself.

I'll make this work like you suggested ;) Thanks for the issue!

**Comment by dimitrisfasoulas** [comment]
Great, thnx for the super fast reply 💯

**Comment by Mastermindzh** [comment]
Implemented it in [1.3.0](https://www.npmjs.com/package/react-cookie-consent/v/1.3.0)

Enjoy :D
